### PR TITLE
138 m5 s05gerentever métricas de prospectos de un agente

### PR DESCRIPTION
**Pull request by Renato549**
**PR Data:**
Task ID: 6200, 6300, 6301, 6400

Problem: Theres not prospect table

Solution: New table component for endpoint-connected lead funnel view

**PR Submitter Checklist:**

Submitter Name: Renato Ramírez y Mónica Ayala

Before marking the PR as ready and asking for somehow to review it, make sure **ALL** steps
are checked. You are responsible for ensuring the answers to all questions are true.

If you make changes to the PR, you **HAVE** to make sure all steps are stil true.

- [x] You added the development minutes to the timelog in the activity plan ?
- [x] All required workflows pass correctly ?
- [x] The branch/PR is in good state
  - [ ] No "137 commits"
  - [ ] No conflicts with `develop`
  - [ ] If you need help, see: https://pastebin.com/tmbHpGDF
- [x] This PR completely marks as finished the related task ?
- [ ] This PR does absolutely no work that isn't directly related to the related task ?
- [x] This PR conforms to the coding standards ?
  - [x] Frontend
    - [x] Components that concern about layout and style, are **100%** pure
- [x] This PR includes automated testing following these points ?
  - [x] Backend
    - [x] New functions that interact with the database have related tests in `__tests__/app/*`
    - [x] New endpoints have related tests in `__tests__/controller/*`
      - [x] Tests for authentication if needed
      - [x] Tests for validation if needed, either for body or search parameters
  - [x] Frontend
    - [ ] Components are rendered using the `RenderTest` fixture at `fixtures.tsx` (see authentication.test.tsx for example)
    - [ ] Test isn't just a render call
- [x] You manually tested any changes that cannot be tested in the automated tests ?
- [ ] If this PR added new features that might be useful for other developers (for example, a new API to send s3 files)
  - [ ] Did you add documentation on the root directory README.md ?
- [x] If this PR needs new data to show how it works on staging
  - [x] Did you add test entities in the seeds function at `seeds/index.ts` ?

**PR Reviewer Checklilst:**

Reviewer Name: Kenny

Before approving and merging the PR, make sure **ALL** steps are checked. You are
responsible for ensuring the answers to all questions are true.

If the submitter makes updates to the PR, you **MUST** go through all steps again.

- [x] You added your name to this description in the "Reviewer name" field.
- [ ] You tested the changes locally
  - [x] You checked out the PR locally
  - [x] You ran the development server
  - [x] You made sure everything starts up correctly
  - [ ] If the automated tests aren't enough, you manually confirmed the changes work as expected
  - [ ] You are sure all changes are either tested manually by you, or in automated tests
- [ ] You reviewd all changes and files manually
  - [ ] Code conforms to coding standards
- [ ] You are sure PR is ready to be merged to staging
- [ ] You are aware merging this PR will result in automatic changes to the staging deployment
- [ ] You marked your test and review minutes in the activity plan
- [ ] You approved and merged the PR

### CODE REVIEW

**Comment by Naiztu** [comment]
You need fill the PR template

**Comment by monica-ayala** [comment]
Checks don't pass, the PR checklist isn't complete and branch is out of date with develop.

File: ram-back/src/controller/prospect.ts [hunk 1]
```diff
@@ -48,3 +50,15 @@ prospectRouter.post(
     res.status(201).json(prospect);
   },
 );
+prospectRouter.get("/count-prospects/:id", async (req, res) => {
+  const db = await getDataSource();
+  const id = req.params.id;
+  // const sales = await db.manager.find(SellEnt);
```
Comment (monica-ayala) [inline ram-back/src/controller/prospect.ts:56]: Delete unused line of code

File: ram-back/src/controller/sale.ts [hunk 2]
```diff
@@ -100,6 +100,7 @@ salesRouter.post(
 
 salesRouter.get("/all", async (req, res) => {
   const db = await getDataSource();
+  // const sales = await db.manager.find(SellEnt);
```
Comment (monica-ayala) [inline ram-back/src/controller/sale.ts:103]: Delete unsued line of code

File: ram-front/index.html
Comment (monica-ayala) [inline ram-front/index.html:1]: Old changes already merged in develop. Make sure PR only contains actual changes and commits made for the task ID.

File: ram-front/src/components/prospects/FunnelTable.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  const { response: agents } = useAxios<IUser[]>({
+    url: `user/all-agents`,
+    method: "GET",
+  });
+
+  return (
+    <div>
+      <div className="mt-8 flex flex-col items-center justify-center">
+        <div
+          data-testid="sales-table"
```
Comment (monica-ayala) [inline ram-front/src/components/prospects/FunnelTable.tsx:21]: data-testid "sales-table"?

File: ram-front/src/pages/ProspectsFunnel.tsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  return (
+    <Wrapper>
+      <div>
+        <div className="flex w-full items-center justify-start pt-8">
+          <h1 className=" rounded-r-2xl bg-gnp-orange-500 py-3 px-20 text-xl font-bold text-white">
+            Prospectos
+          </h1>
+        </div>
+        <div className="mt-8 flex flex-col items-center justify-center">
+          {/* <SalesTable sales={sales} onDeleted={() => updateSales()} /> */}
```
Comment (monica-ayala) [inline ram-front/src/pages/ProspectsFunnel.tsx:19]: Delete unused code

**Review by monica-ayala — CHANGES_REQUESTED** [review]

**Comment by monica-ayala** [comment]
Avoid having 137 commits: https://pastebin.com/tmbHpGDF

**Review by monica-ayala — APPROVED** [review]

**Review by eduardvercaemer — APPROVED** [review]

**Pull request merged**